### PR TITLE
PR for Issue 19721: Externalize authorizationHeaderScheme MP JWT config attribute

### PR DESCRIPTION
**Pull request by ayoho**
For #19721

### CODE REVIEW

**Comment by ayoho** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.1,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-2.0,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-2.0

**Review by helyarp — COMMENTED** [review]
Looks good!

**Comment by ayoho** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.1,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.1_fat.mpJwt-2.0,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-1.2,com.ibm.ws.security.mp.jwt.1.2_fat.mpJwt-2.0

File: dev/com.ibm.ws.security.mp.jwt/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -57,6 +57,9 @@ audiences.desc=The trusted audience list to be included in the aud claim in the
 clockSkew=The time difference allowed between systems
 clockSkew.desc=This is used to specify the allowed clock skew in minutes when validating the JSON web token.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.mp.jwt/resources/OSGI-INF/l10n/metatype.properties:58]: Not sure if you can change this at this time but if so:

This is used to specify the allowed clock skew in minutes when validating the JSON web token.
--->
This attribute specifies the allowed clock skew in minutes when the JSON web token is validated.

**Review by dmuelle — APPROVED** [review]

**Comment by skasund** [comment]
L2 message review not required

**Pull request merged**